services/movie.py

```diff
@@ -1,3 +1,4 @@
+from django.db import transaction
```

Comment:
Consider using double quotes for consistency with the code style guideline that prefers double quotes.